### DISCUSSION TITLE
Preview out, RC in (now in third version)...

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-Install#54] [General]
[Windows Package Manager](https://github.com/microsoft/winget-cli/releases)
Soon...

**Comment by Romanitho** [comment]
?

**Reply by KnifMelti** [reply]
Just glad that it soon vill be rolling out via Store 👍

**Reply by Romanitho** [reply]
They already published a v1.3 no ?

**Reply by KnifMelti** [reply]
Via Store? No, it's the third Release Candidate now what I can tell. Hence the soon...

**Reply by Romanitho** [reply]
From the store, App Installer version 1.18.1741.0 is available

**Reply by KnifMelti** [reply]
Downgraded to the old https://github.com/microsoft/winget-cli/releases/tag/v1.2.10271 (marked as Latest) which is **1.17.10.271.0**.
Started Store, run an upgrade from it and ended up at **Windows Package Manager v1.2.11601**, which is **1.17.11601.0** and the only existing in Store (from Sweden).
So, soon...

**Reply by Romanitho** [reply]
ah very true, it was my previous RC ^^
